monorepo-trigger/pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - label: ":git: Check changed files"
```

Comment:
Looking at this in the visualiser, I think something like `Conditionally trigger pipelines` might be more informative.

Reply:
It could also be referred to as doing a **diff**?

Reply:
Or "Run against changed files"